src/modules/main/navbar/hooks/useWindowSize.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+
+export interface Size {
+  width: number;
+  height: number;
+}
```

Comment:
This interface  should be wrapped by namespace I think.

Reply:
Only components/classes should use namespaces